esignet-service/scripts/client-mgmt-smoke.sh

```
@@ -0,0 +1,207 @@
+		}
+		printf 'Authorization: Bearer %s' "$token"
+		return 0
+	fi
+	return 0
+}
+
+run_client_mgmt_smoke() {
+	echo "Client management smoke — ${BASE_URL}"
+	echo ""
+
+	local auth_header client_id suffix create_body create_resp get_resp update_body update_resp
```

Shellcheck (0.11.0)

[warning] create_resp appears unused. Verify use (or export if used externally).

(SC2034)